src/anyio/_backends/_asyncio.py

```diff
@@ -1674,6 +1676,8 @@ def _create_task_info(task: asyncio.Task) -> TaskInfo:
 
 
 class TestRunner(abc.TestRunner):
+    _send_stream: MemoryObjectSendStream[tuple[Awaitable[Any], asyncio.Future[Any]]]
```

Comment:
In the trio implementation, I do a null check against it so I needed to set a value. It felt natural to do that in `__init__()` since I don't have any other class annotations there. Just a matter of taste I suppose.

Reply:
I have a preference for keeping instance variables in `__init__` and declaring class variables at the top level, but I think whatever your preference is here is still plenty clear.